src/components/Forgot/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { Form, Input, Button, Alert } from "antd";
+import firebase from "../../firebaseConfig";
+import { MailOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+
+function Forgot({ isTesting }) {
+  const { t } = useTranslation();
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  const forgotPassword = (e) => {
+    setErrorMessage(null);
+    firebase
+      .auth()
+      .sendPasswordResetEmail(email)
+      .then(function (user) {
+        alert("Please check your email...");
+      })
+      .catch((err) => {
+        setErrorMessage(err.message);
+      });
+  };
+
+  const layout = {
+    labelCol: { span: 4 },
+    wrapperCol: { span: 8 },
+  };
+
+  const tailLayout = {
+    wrapperCol: { offset: 8, span: 16 },
+  };
+
+  return (
+    <Form {...layout} {...tailLayout}>
+      {errorMessage ? (
+        <Alert
+          message="You Can start add your coupon"
+          description="Success"
+          type="success"
+        />
+      ) : null}
+      {!errorMessage ? <p>{errorMessage}</p> : null}
+      <Form.Item
+        rules={[{ required: true, message: "Please input your username!" }]}
+        value={email}
+        onChange={(e) => setEmail(e.target.value)}
+      >
+        <Input
+          prefix={<MailOutlined />}
+          style={{ width: "40em", display: "flex" }}
+          placeholder={t("forgot.title.title1")}
+          action="click"
+          type="email"
+          className="ant-input"
+          value={email}
+          onChange={(e) => setEmail(e.target.value)}
```

Comment:
im confused here why there's 2 inputs that are getting the user email ?

Reply:
I forgot to delete this line now fixed.